src/reducer.ts

```diff
@@ -1,7 +1,7 @@
-import { mutate } from "./mutate"
-import { payloadError } from "./errors"
+import {mutate} from "./mutate"
+import {payloadError} from "./errors"
 
-import type { Action, State } from "./types"
+import type {Action, State} from "./types"
```

Comment:
Would you mind reverting the formatting changes for these imports?

Reply:
Ah my bad. Fixed!